### ISSUE TITLE
Merge commands

### ISSUE THREAD

**Issue by pingles** [uswitch/kiam#98]
This will help reduce the size of the published Docker image.

There are 3 separate binaries that get built:

* server
* health
* agent

The docker image contains the three binaries which will mostly be copies of exactly the same code.

Rather than having separate binaries we should join them together and make them subcommands.

**Comment by DewaldV** [comment]
Recommend we ensure the old names are still available in the image. IE running `agent ...` in the image should still result in an agent running maybe via `kiam agent ...`

We might not want to require everybody to update their yaml in the next release. We could support the aliases until v3.

**Comment by pingles** [comment]
I think there'll likely be breaking changes in v3 anyway so I'd suggest we just change the commands and require updated YAML.

**Comment by sambooo** [comment]
Users should be able to bump minor versions without updating the `command` for their pods, so unless we've decided to not do another 2.x release we need the aliases. Has it been decided that 3.0 is the next version?

**Comment by pingles** [comment]
Yep, no more 2.x. There's been TLS changes that change behaviour so it'll
be a 3.0 next.

**Comment by ewbankkit** [comment]
I can make the relevant changes to the [Helm chart](https://github.com/kubernetes/charts/tree/master/stable/kiam) when the time is right.
